utils.h

```diff
@@ -28,13 +28,14 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <syslog.h>
```

Comment:
stupid minor nit: I think this actually goes *above* `unistd.h`? the ordering seems to be single-component paths, in alphabetical order, then multi-component paths. (a lot of the includes elsewhere aren't ordered consistently though so idk if that matters)

Reply:
Yeah, let's at least make this consistent(ish) in new code.